### ISSUE TITLE
Projektname in der App anzeigen

### ISSUE THREAD

**Issue by SlevinL** [SlevinL/DeploymentManager#13]
Derzeit wird nur der letzte Teil des Images bei einer APP angezeigt. Wird aber ein Image mehrfach als unterschiedliche APP deployed, kann man keinen Unterschied mehr erkennen. 
Daher sollte in der APP fee Projektname angezeigt werden. Vielleicht die entity App um das Attribut erweitern.

**Comment by SlevinL** [comment]
@UKirsche

**Comment by UKirsche** [comment]
Könntest du das mit Beispiel beschreiben? Es geht um die Stages-Darstellung. Dort muss eine zusätzliche Info rein? Vielleicht kleiner Screenshot mit Darstellungsvorschlag?

**Comment by SlevinL** [comment]
Ja, man kann Projekte verbinden. Das bedeutet, das verschiedene Projekte die gleichen Images verwenden. Konkret geht es um den dms Service. Dieser wird als mgs und dmp service deployed. In der Stage sieht man aber zwei mal dmsservice, da das Image das gleiche ist. Es sind aber zwei verschiedene Projekte.

**Comment by UKirsche** [comment]
Was findest du besser? Mir gefällt die Tooltip-Variante besser:
<img width="191" alt="Bildschirmfoto 2020-04-15 um 11 54 25" src="https://user-images.githubusercontent.com/1540036/79324788-6c377700-7f10-11ea-8be9-996779c71bde.png">
<img width="166" alt="Bildschirmfoto 2020-04-15 um 11 57 02" src="https://user-images.githubusercontent.com/1540036/79324804-722d5800-7f10-11ea-87e6-f28165e8766a.png">

**Comment by SlevinL** [comment]
Hallo, den Tooltip würde ich nicht nehmen, da man dann nicht auf den ersten Blick erkennen kann, um welches Projekt es sich handelt. 
Am besten wäre der Projektname und das Image vielleicht in den Tooltip.

**Comment by UKirsche** [comment]
Diese Variante sieht gut aus, hab mehrere durch probiert:
<img width="143" alt="Bildschirmfoto 2020-04-15 um 13 15 19" src="https://user-images.githubusercontent.com/1540036/79331335-482d6300-7f1b-11ea-906a-dd09d76b028b.png">

Was meinst?

**Comment by UKirsche** [comment]
Soll das Feld project_name aus der AppEntity da rein?

**Comment by SlevinL** [comment]
Ja stimmt der ist ja schon da.

**Comment by UKirsche** [comment]
Sieht nu so aus:

<img width="257" alt="Bildschirmfoto 2020-04-15 um 14 40 33" src="https://user-images.githubusercontent.com/1540036/79338281-4ff30480-7f27-11ea-9ba7-c32ff9143380.png">
